roles/container-engine/docker/tasks/reset.yml

```diff
@@ -6,8 +6,8 @@
 
 - name: Docker | Find docker packages
   set_fact:
-    docker_packages_list: "{{ ansible_facts.packages.keys() | select('search', '^docker*') }}"
-    containerd_package: "{{ ansible_facts.packages.keys() | select('search', '^containerd*') }}"
+    docker_packages_list: "{{ ansible_facts.packages.keys() | select('search', '^docker*') | list }}"
```

Comment:
This should not be needed since we dropped ansible 2.9 and 2.10, generators are automatically typecast to lists starting with ansible 2.11.

Reply:
Thanks, I was not aware of this

I'm removing this part